library/aws/autoscale-deployment.ts

```diff
@@ -357,7 +358,7 @@ export function createLoadBalancer(tfgen: TF.Generator, tfname: string, sr: shar
     };
     const lb = AR.createLb(
       tfgen,
-      'alb',
+      params.alb_name || 'alb',
       applyCustomize(params.customize_lb, lbParams)
     );
   const lb_http_listener = AR.createLbListener(tfgen, tfname + '_http', {
```

Comment:
You're going to need unique names for all the resources created by the pattern, not just the load balancer itself.
ie:
createLb, createLbListener (http), createLbListener (https).

Reply:
Other resources are already using `tfname` as a base.